tests/api/core/strapi/document-service/publish.test.api.ts

```diff
@@ -24,6 +24,19 @@ describe('Document Service', () => {
   });
 
   describe('Publish', () => {
+    testInTransaction('Publishing keeps creator fields from the draft', async () => {
+      const articleDb = await findArticleDb({ title: 'Article1-Draft-EN' });
+
+      const publishedArticle = await publishArticle({
```

Comment:
Just so we separate testing concerns could you do a publish without populate & then a find with populate. 

This way we "ensure" it doesn't work by side-effect of the manual populate